### DISCUSSION TITLE
Looking to move from tt-rss, few questions

### DISCUSSION THREAD

**Discussion by wrobelda** [FreshRSS/FreshRSS#3540] [Q&A]
I searched but couldn't find or wasn't sure of a definitive answer:

1) is it possible to define regex filters?
2) is there a notification plugin for puhover/pushbullet/telegram, or any other instantaneous push mechanism?
3) is it possible to chain filter and a plugin's action, such that the notification would only be sent only for the unfiltered articles?
4) is it possible to define a custom refresh interval per feed? I need a refresh interval of as little as 60s for my custom, private feeds. I [saw](https://github.com/FreshRSS/FreshRSS/issues/1158) that there's a workaround, but that would apply to all the feeds, including the public ones, which would surely result in IP bans.

**Comment by Frenzie** [comment]
There's a global update frequency and individual settings per feed.

But since you say custom & private, have you considered adding WebSub?

**Reply by wrobelda** [reply]
Thanks. Yes, I am aware of WebSub, but really it's a hack project that just works (and has for many years), and the 60s interval is good enough.

**Comment by Alkarex** [comment]
Regex filters https://github.com/FreshRSS/FreshRSS/pull/6706
